packages/server/api/services/Themization.ts

```diff
@@ -0,0 +1,51 @@
+import express, { type Request, type Response, Router } from 'express';
+
+import { checkAuthMiddleware } from '../../middlewares';
+import { Themes } from '../../models/Themes';
+import { UserThemes } from '../../models/UserThemes';
+
+export const themizationRoute = Router()
+  .use(express.urlencoded({ extended: true }))
+  .use(express.json())
+  .use('/', checkAuthMiddleware)
+  .get('/:userId', async (req: Request, res: Response): Promise<Response> => {
+    const { userId } = req.params;
+    /** Данные юзера из checkAuthMiddleware*/
+    const authUserId = res.locals.user.id;
+
+    if (authUserId !== userId) {
```

Comment:
Здесь супер, теперь не взломают.

Только у меня typeof authUserId - number, а typeof userId  - string.

Я бы поменял на что-то вроде

```suggestion
    if (+authUserId !== +userId) {
```